src/meshes/mesh_io.jl

```diff
@@ -119,6 +120,29 @@ function save_mesh_file(mesh::StructuredMesh, output_directory; system = "")
     return filename
 end
 
+# Does not save the mesh itself to an HDF5 file. Instead saves important attributes
+# of the mesh, like its size and the type of boundary mapping function.
+# Then, within Trixi2Vtk, the StructuredMesh and its node coordinates are reconstructured from
+# these attributes for plotting purposes.
+function save_mesh_file(mesh::StructuredMeshView, output_directory; system = "",
+                        timestep = 0)
+    # Create output directory (if it does not exist)
+    mkpath(output_directory)
+
+    filename = joinpath(output_directory, @sprintf("mesh_%s_%06d.h5", system, timestep))
```

Comment:
To clarify: the time step is stored in the filename as the mesh might change over time over the course of a simulation, correct?

Reply:
Correct. Without it we would end up with only one mesh file for all snapshots. But our meshes can change in time and therefore need different mesh files.

Reply:
Thanks for the clarification! /r